src/test/java/Collections/MyArrayListTest.java

```diff
@@ -0,0 +1,72 @@
+package Collections;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class MyArrayListTest {
```

Comment:
The test should be testing different angles of the logic. Null cases, empty cases, etc.